client/src/lib/components/OrganizationCard/OrganizationCard.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  export let name: string;
+  export let src: string;
+  export let status: string;
+</script>
+
+<div class="fr-card">
+  <div class="fr-card__body">
+    <div class="fr-card__content">
+      <h3 class="fr-card__title">
+        {name}
+      </h3>
+      <div class="fr-card__start">
+        {#if status == "catalog"}
+          <p class="fr-badge fr-badge--success">catalogue Activé</p>
```

Comment:
(Détail) Ça n'impacte pas le rendu car `fr-badge` met tout en majuscules, mais :

```suggestion
          <p class="fr-badge fr-badge--success">Catalogue activé</p>
```